src/setup-signaling.sh

```diff
@@ -379,7 +385,13 @@ function signaling_step5() {
 	deploy_file "$TMP_DIR_PATH"/signaling/nginx-signaling-upstream-servers.conf /etc/nginx/snippets/signaling-upstream-servers.conf || true
 	deploy_file "$TMP_DIR_PATH"/signaling/nginx-signaling-forwarding.conf /etc/nginx/snippets/signaling-forwarding.conf || true
 
-	deploy_file "$TMP_DIR_PATH"/signaling/janus.jcfg /etc/janus/janus.jcfg || true
+	if [ "$(dpkg --print-architecture)" = "arm64" ]; then
+		deploy_file "$TMP_DIR_PATH"/signaling/janus_aarch64.jcfg /etc/janus/janus.jcfg || true
+	elif [ "$(dpkg --print-architecture)" = "ppc64el" ]; then
+		deploy_file "$TMP_DIR_PATH"/signaling/janus_powerpc64le.jcfg /etc/janus/janus.jcfg || true
```

Comment:
Please remove the janus_powerpc64le.jcfg again, we don't have a use case for this as we don't support ppc64le as an architecture. Or do we?

Reply:
I removed support for PowerPC64LE